apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -154,7 +154,11 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
         }
         return Promise.resolve(true)
     }
-
+    const validateDate = (str) =>  {
```

Comment:
Function name does't corresponds to what it actually does.
Use it for validation only.
In place, where dates are actually passed to GraphQL input variables, you can format it.

Reply:
Renamed